beakerx/README.md

```diff
@@ -1,3 +1,5 @@
 Select your DC/OS version:
 
 [1.10](1.10)
+
```

Comment:
I don't think we have blank lines in the other Readme's

Reply:
Looking e.g. at https://github.com/dcos/examples/blob/master/arangodb/README.md I see a blank line between the versions